lib/commands/update.js

```diff
@@ -179,6 +181,25 @@ class UpdateCommand extends Command {
         });
     }
 
+    rollbackFromFail(error, argv) {
+        const previousVersion = this.system.getInstance().cliConfig.get('previous-version');
+        this.ui.error(error);
+        const question = argv.version === '2.0.0' ?
```

Comment:
Not sure we need to differentiate the text here 🤔 What was your thinking? :)

Reply:
It was a mix of abstracting the utility of the function and a more explicit message about 2.0

I originally wrote the snippet based on the logic of only rolling back if upgrading to 2.x fails, so the error was explicit in saying `Ghost 2.0`. As I was creating the PR I realized this is applicable to all updates, not just to 2.0

With specifically saying 2.0 I thought it would be useful as a reminder since it's a major update. Now that I think about it, it might be useful to specify the attempted version regardless of the semver type

Reply:
Yeah would be nice to just show the version which has failed.

e.g. `Unable to upgrade from X to Y. Would you like to revert to X?`

Reply:
Thanks 👍